service kwServer{    
    string get_keyword(1:string text, 2:string phrase, 3:i32 num) 
}    
